### PR TITLE
Use AlertDialog

### PR DESCRIPTION
**Pull request by RobozinhoD**
## Purpose / Description
_Describe the problem or feature and motivation_

## Fixes
Related #13315

## Approach
use AlertDialogBuilder

## How Has This Been Tested?

I opened the dialogs I edited. But I didn't know how to test the dialog on IntroductionActivity

CardTemplateBrowserAppearanceEditor
image

BackendImporting
image

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/IntroductionActivity.kt [hunk 1]
```diff
@@ -113,11 +117,11 @@ class IntroductionActivity : AppIntro() {
     // TODO: Factor this into the AppLoadedFromBackupWorkaround class
     private fun handleStartupFailure(startupFailure: StartupFailure) {
         if (startupFailure == StartupFailure.WEBVIEW_FAILED) {
-            MaterialDialog(this).show {
+            AlertDialog.Builder(this).show {
                 title(R.string.ankidroid_init_failed_webview_title)
                 message(R.string.ankidroid_init_failed_webview, AnkiDroidApp.webViewErrorMessage)
                 positiveButton(R.string.close) { finish() }
-                cancelable(false)
+                setCancelable(false)
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/IntroductionActivity.kt:124]: please add this as an extension method on `AlertDialogFacade.kt`

**Review by david-allison — CHANGES_REQUESTED** [review]
Love how well the facade turned out. One nitpick to make it nicer

File: AnkiDroid/src/main/java/com/ichi2/anki/IntroductionActivity.kt (hunk 1 above)
Comment (RobozinhoD) [reply AnkiDroid/src/main/java/com/ichi2/anki/IntroductionActivity.kt]: ok

**Review by david-allison — APPROVED** [review]
Final commit isn't named correctly.
Could you confirm whether `cancelable` both stops the back button and cancelling by tapping outside?
LGTM otherwise

**Review by BrayanDSO — APPROVED** [review]
Dialogs war, second battle finished

**Pull request merged**